openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx

```diff
@@ -142,6 +171,21 @@ const SampleDataTable = ({ isTableDeleted, tableId }: SampleDataProps) => {
       })}
       data-testid="sample-data"
       id="sampleDataDetails">
+      <Space className="m-b-md justify-end w-full">
+        <Tooltip title={!hasPermission && NO_PERMISSION_FOR_ACTION}>
+          <Button
+            danger
+            data-testid="delete-sample-data"
+            disabled={!hasPermission}
```

Comment:
In other places, we are not showing the button itself if there is no permission, instead of a disabled button. Can we follow the same here?